geometry/bbox.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include <limits>
+#include <vector>
+
+namespace m2
+{
+class BBox
```

Comment:
How is this class different from m2::RectD?

Reply:
Bounding box is a bounding box around some set of points, RectD is a rectangle, not related to points on the plane at all.